Polyfills/XMLHttpRequest/Source/XMLHttpRequest.cpp

```diff
@@ -77,12 +80,16 @@ namespace Babylon::Polyfills::Internal
                 InstanceAccessor("responseType", &XMLHttpRequest::GetResponseType, &XMLHttpRequest::SetResponseType),
                 InstanceAccessor("responseURL", &XMLHttpRequest::GetResponseURL, nullptr),
                 InstanceAccessor("status", &XMLHttpRequest::GetStatus, nullptr),
+                InstanceAccessor("ok", &XMLHttpRequest::Ok, nullptr),
+                InstanceAccessor("headers", &XMLHttpRequest::GetAllResponseHeaders, nullptr),
                 InstanceMethod("getResponseHeader", &XMLHttpRequest::GetResponseHeader),
                 InstanceMethod("addEventListener", &XMLHttpRequest::AddEventListener),
                 InstanceMethod("removeEventListener", &XMLHttpRequest::RemoveEventListener),
                 InstanceMethod("abort", &XMLHttpRequest::Abort),
                 InstanceMethod("open", &XMLHttpRequest::Open),
                 InstanceMethod("send", &XMLHttpRequest::Send),
+                InstanceMethod("setRequestHeader", &XMLHttpRequest::SetRequestHeader),
+                InstanceMethod("json", &XMLHttpRequest::Json),
```

Comment:
Again, not in the spec. Are you seeing this somewhere? I'm not sure why these non-spec things are being added.